task_9/get-props-longer-than-5.js

```diff
@@ -0,0 +1,11 @@
+function getPropsLongerThan5(obj) {
+  const niz = [];
+  for (const variable in obj) {
+    if (variable.length > 5) {
+      niz.push(variable);
+    }
+  }
+  return niz;
+}
+
```

Comment:
for in metoda moze da bude ok resenje, ali se u principu vise ne koristi i smatra losom praksom zato sto kada radis for in on ce traziti ne samo property-je tog objekta vec i svih objekta u njegovom prototype lancu ako ih ima
https://stackoverflow.com/questions/500504/why-is-using-for-in-with-array-iteration-a-bad-idea
Ako vec radis na ovaj nacin treba ubaciti proveru da li property pripada tom objektu sa `hasOwnProperty()`
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/hasOwnProperty
ili po meni bolji nacin da koristis `Object.keys()` koji vraca samo own propertije
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/keys
```suggestion
const properties = Object.keys(obj);
properties.forEach(prop => {
  if(prop.length > 5) {
    niz.push(prop)
  }
}
```